Sources/xcproj/PBXObject.swift

```diff
@@ -1,27 +1,19 @@
 import Foundation
 
 /// Class that represents a project element.
-public class PBXObject: Referenceable, Decodable {
-
-    public var hashValue: Int { return self.reference.hashValue }
-
-    /// Element unique reference.
-    public let reference: String
-
-    init(reference: String) {
-        self.reference = reference
-    }
+public class PBXObject: Decodable {
 
+    // MARK: - Init
+    
+    init() {}
```

Comment:
This class should not actually be initiable as it's abstract, so we can remove this

Reply:
Good point, removing.

Reply:
Ah ok. I guess it’s not public anyway so not a big issue